src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// Write code here
+/* eslint-disable */
+import React from 'react';
+import '../../App.scss';
+
+export default function Person(props) {
+  const personPartner = props.sex === 'm' ? 'wife' : 'husband';
+  const marriageStatus = props.isMarried
+    ? `My ${personPartner}'s name is ${props.partnerName}`
+    : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+       { `My name is
+        ${props.name}`}
```

Comment:
why not single-line?
```suggestion
       { `My name is ${name}` }
```

if you multi-line like that - make sure block brackets (square, round or curly) are properly indented and new-lined:
```suggestion
       {
         `My name is
         ${name}`
       }
```